pinboard/pinboard.py

```diff
@@ -172,9 +172,12 @@ def __call__(self, *args, **kwargs):
                 401: exceptions.PinboardAuthenticationError,
                 403: exceptions.PinboardForbiddenError,
                 500: exceptions.PinboardServerError,
+                503: exceptions.PinboardServiceUnavailable,
             }
-            Error = error_mappings[e.code]
-            raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            if e.code in error_mappings:
+                Error = error_mappings[e.code]
+                raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            raise
```

Comment:
As a user of the library, I don't care if the library is handling the error or not -- I'm catching the base class anyway so I only need to have one exception clause. The true problem is that sometimes the original code raised a KeyError and sometimes it raised an error derived from urllib2.HTTPError, and that made it more difficult to use the library. By re-raising the existing error the change does indicate that the error type is not considered special to the pinboard library.